run-tests.sh

```
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Try to install composer dev dependencies
+composer install --no-interaction --no-scripts --no-plugins
+
+# If that failed, exit.
+rc=$?; if [[ $rc != 0 ]]; then exit $rc; fi
+
+# Run the feature tests
+vendor/bin/behat --stop-on-failure
```

Could this be updated to use `runny`? Same with the `composer install` line, above.

---

we need runny for commands that bash will ignore errors on and proceed to next command, but the last command in the file does not need runny so we can see its full output and its exit code will stop a build if it fails anyway

---

Oh, does the return value / output of the last command in a bash script end up being the return value / output when the bash script itself is run? If so, is there anything else runny would gain us that is worth having here (like logging to syslog)?